cloud/terraform/aws/backup/modules/backup-plan/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_backup_plan" "liferay-backup" {
+	dynamic "rule" {
+		content {
+			lifecycle {
+				delete_after=rule.value.retention_days
+			}
+			rule_name=rule.value.rule_name
+			schedule=rule.value.schedule
+			start_window=rule.value.start_window_minutes
+			target_vault_name=var.backup_vault_name
+		}
+		for_each=var.backup_rules
+	}
+	name=var.backup_plan_name
+	tags={
+		DeploymentID=var.unique_deployment_id
+	}
+}
+resource "aws_backup_selection" "by_backup_tag" {
+	iam_role_arn=var.backup_service_assumed_role_arn
+	name=var.backup_selection_name
+	plan_id=aws_backup_plan.liferay-backup.id
+	selection_tag {
+		key="Backup"
+		type="STRINGEQUALS"
+		value="true"
+	}
+	selection_tag {
+		key="DeploymentID"
```

Comment:
Added an additional selection tag so that we don't backup from multiple environments or applications.